NetKAN/RealBattery.netkan

```diff
@@ -0,0 +1,16 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "RealBattery",
+    "$kref"        : "#/ckan/github/blackliner/RealBattery",
+    "license"      : "MIT",
+    "ksp_version"  : "1.2",
```

Comment:
You have an AVC .version file, so you could replace this line with the following and KSP compatibility on CKAN will track with AVC:

    "$vref"        : "#ckan/ksp-avc",